docs/development/translations.rst

```diff
@@ -22,6 +28,13 @@ about anything in this document, here's how to ask for help:
 
 * Post a message in the `translations category of the SecureDrop forum`_
 * Chat in the `SecureDrop instant messaging channel`_
+
+   * Localization Lab, with whom we :ref:`coordinate SecureDrop's
+     translation<when_is_securedrop_translated>`, also maintains `their own
```

Comment:
I think this link is a bit confusing because it takes you to _when_ translations can happen, which felt more low level than how we coordinate with Localization Lab. Also this is in the context of "Getting help" so I think pointing to their channel and who they are would be enough for this section.

Reply:
Fair!  Simplified in 6ac2ad8d6c26c0dd6edd7e69cfe066bef85fa588.